containers/python-3-pypy/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# [Choice] Pypy docker images. Choices are 2,3. Complete list of tags is avaiable at: https://hub.docker.com/_/pypy
```

Comment:
@Cyb3r-Jak3 This is not a valid format for the [Choice] argument since this drives UX. A valid example would be:

```suggestion
# [Choice] Pypy version: 2, 3
```
...which would then allow users to pick when adding this definition from VS Code.

If you don't want to maintain a list, you can remove the `[Choice]` prefix.

Reply:
Fixed